src/components/Invoice/InvoiceForm.tsx

```diff
@@ -44,7 +45,9 @@ export default function InvoiceForm({ defaultValues }: InvoiceProps) {
         <form onSubmit={handleSubmit(onSubmit)}>
           <Grid item container>
             <Grid item xs={6} container>
-              <TextField {...register('id')} fullWidth label="No." variant="standard" error={!!errors?.id} />
+              {!isNew && (
```

Comment:
I don't think you need this. Just pass undefined as id when data is empty.

Reply:
done